File: Java/백준/Bronze/1100. 하얀 칸/하얀 칸.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int sum=0;
        for (int i =0; i<8; i++){
            String a = in.next();
            for(int j=0; j<4; j++){
                if(a.charAt(j*2+i%2)=='F'){
                    sum++;
                }
            }
        }
        System.out.println(sum);
    }
}

File: C#/백준/Bronze/1100. 하얀 칸/하얀 칸.cs
using System;

namespace q{
    class w{
        static void Main(){
            int sum=0;
            for(int i=0; i<8; i++){
                string a = Console.ReadLine();
                for(int j=0; j<4; j++){
                    if(a[j*2+i%2]=='F'){
                        sum++;
                    }
                }
            }
            Console.WriteLine(sum);
        }
    }
}